generate/codegen/gen_struct.go

```diff
@@ -0,0 +1,19 @@
+package codegen
```

Comment:
i think since you originally submitted this i added a struct generator and we should be generating all of them for the frameworks we have: https://github.com/progrium/darwinkit/blob/main/generate/tools/structs.go

Reply:
Oh great, thanks, happy to cull this.